readme.md

```diff
@@ -188,6 +183,47 @@ mbo.finalize().then(function(mediaBlobs) {
 });
 ```
 
+### Error Handling in finalize
+When *[finalize()](#finalize-method)* is called, the User Agent will perform these basic checks for the operations that are batched. This error checking should be done before executing any of the operations.
+
+For *[trim()](#trim-method)*
+1. Let *O* represent the blob to be trimmed
+2. If *startTime* is less than 0 **OR** *endTime* is greater than the *O*.duration **OR** *startTime* is greater than the *endTime*:
+    * create a new [ErrorEvent](https://html.spec.whatwg.org/multipage/webappapis.html#errorevent)
```

Comment:
I think we want to reject the finalize promise with a DomException instead of using an event: https://heycam.github.io/webidl/#idl-exceptions.  Here's an example to follow: https://w3c.github.io/ServiceWorker/#resolve-get-client-promise-algorithm

Reply:
What about other errors we throw ? Like in the constructor for MediaBlob ?
Should those also be DomExceptions ?